apps/drec-api/src/pods/issuer/issuer.service.ts

```diff
@@ -0,0 +1,172 @@
+import { Injectable, NotFoundException, Logger, Inject } from '@nestjs/common';
+import { Cron, CronExpression } from '@nestjs/schedule';
+import {
+  CertificateService,
+  CERTIFICATE_SERVICE_TOKEN,
+  IIssueCommandParams,
+} from '@energyweb/origin-247-certificate';
+import { CertificateSourceType, ICertificateMetadata } from '../../utils/types';
+import { DateTime } from 'luxon';
+import {
+  FilterDTO,
+  ReadsService as BaseReadsService,
+} from '@energyweb/energy-api-influxdb';
+import { DeviceService } from '../device/device.service';
+import { DeviceGroup, IDeviceGroup } from '../device-group/device-group.entity';
+import { Device, IDevice } from '../device/device.entity';
+import { BASE_READ_SERVICE } from '../reads/const';
+import { OrganizationService } from '../organization';
+import { DeviceGroupService } from '../device-group/device-group.service';
+import { ConfigService } from '@nestjs/config';
+
+@Injectable()
+export class IssuerService {
+  private readonly logger = new Logger(IssuerService.name);
+
+  constructor(
+    private groupService: DeviceGroupService,
+    private deviceService: DeviceService,
+    private organizationService: OrganizationService,
+    @Inject(CERTIFICATE_SERVICE_TOKEN)
+    private readonly certificateService: CertificateService<ICertificateMetadata>,
+    @Inject(BASE_READ_SERVICE)
+    private baseReadsService: BaseReadsService,
+    private readonly configService: ConfigService,
+  ) {}
+
+  // @Cron(CronExpression.EVERY_10_SECONDS)
```

Comment:
hm?

Reply:
I left this in case we need to troubleshoot it later on and don`t want to wait until 23:30 :D